<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>681</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>clinical observation labels from bipoles  - NOT FOR MEDICAL USE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>681</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="lowHz_radiobutton">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1 Hz</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">stim_selection_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="highHz_radiobutton">
        <property name="text">
         <string>50 Hz</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">stim_selection_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bothHz_radiobutton">
        <property name="text">
         <string>Both</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">stim_selection_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="fontColor_radiobutton">
        <property name="text">
         <string>FontColor (Physiological or Seizure Type)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">excelcolor_selection_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="backColor_radiobutton">
        <property name="text">
         <string>BackColor (je sais pas comment appeler Ã§a)</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">excelcolor_selection_group</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListView" name="list_backColor_Condition">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="stim_selection_group"/>
  <buttongroup name="excelcolor_selection_group"/>
 </buttongroups>
</ui>
